autopush/settings.py

```diff
@@ -74,9 +75,18 @@ def __init__(self,
         self.storage = Storage(self.storage_table)
         self.router = Router(self.router_table)
 
+        # CORS
+        self.cors = enable_cors
+
     def update(self, **kwargs):
         for key, val in kwargs.items():
             if key == "crypto_key":
                 self.fernet = Fernet(val)
             else:
                 setattr(self, key, val)
+
+    def get(self, key):
+        """ cyclone/web.py is calling self.settings.get("debug") where
+        self == EndpointHandler. this is causing an unhandled exception.
```

Comment:
It is? If cyclone already attaches its own 'settings' object to the handler, we'll need to rename ours to keep them separate.